master/getting-started/rkt/installation/manual.md

```diff
@@ -98,7 +95,7 @@ b52bba11  node  quay.io/calico/node:latest  running 10 seconds ago  10 seconds a
 
 ## Intalling the calicoctl CLI tool
```

Comment:
And now that I look at it, i don't think the numbers look good on the list of components.